src/test/java/com/epam/ld/module2/testing/template/TemplateEngineTest.java

```diff
@@ -1,4 +1,38 @@
 package com.epam.ld.module2.testing.template;
 
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
 public class TemplateEngineTest {
+
+    @Test
+    public void checkGenerateMessageWithValidTemplateFromConsole() {
+        fail();

```

Comment:
Usually when we write test with TDD we actually implement tests fully, but the implementation of the production classes is missing.